resources/ext/build-scripts/gem-list.txt

```diff
@@ -5,3 +5,4 @@ locale 2.1.2
 gettext 3.2.2
 gettext-setup 0.28
 fast_gettext 1.1.0
```

Comment:
should this version be updated or will that come from a merge up?

Reply:
It should probably come from the merge-up, but we should also update to 1.1.2 in the 5.1.x branch.